### PR TITLE
[CP-1951] [Relaxation] Audio crashes when user uploads the same file as being played by relaxation app

### PR DESCRIPTION
**Pull request by lkowalczyk87**
Jira: [CP-1951]

**Description**

<details>
<summary><b>Screenshots</b></summary>

image

image

</details>

[CP-1951]: https://appnroll.atlassian.net/browse/CP-1951?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/app/src/files-manager/components/upload-files-modals/upload-files-modals.component.tsx [hunk 1]
```diff
@@ -82,15 +83,17 @@ export const UploadFilesModals: FunctionComponent<UploadFilesModalProps> = ({
...
-        <ErrorModal
-          testId={UploadFilesModalsTestIds.ErrorModal}
-          open={uploadingFailed}
-          title={intl.formatMessage(errorTitle)}
-          subtitle={intl.formatMessage(errorSubtitle)}
-          closeModal={onCloseUploadingErrorModal}
-        />
-      )}
+      {uploadingFailed &&
+        error?.type !== FilesManagerError.UploadDuplicates && (
```
Comment (dkarski) [inline packages/app/src/files-manager/components/upload-files-modals/upload-files-modals.component.tsx:87]: what if the type of error is another one? The application displays some error or can't be happened right now?

Comment (lkowalczyk87) [reply packages/app/src/files-manager/components/upload-files-modals/upload-files-modals.component.tsx:87]: All other error types are shown with a general error modal, as it has been so far

File: packages/app/src/files-manager/components/duplicated-files-modal/duplicated-files-modal.component.tsx [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+          message={messages.duplicatedFilesUploadModalListTitle}
+        />
+        <DuplicatedFilesListItems>
+          {duplicatedFiles.map((fileName, index) => {
+            return (
+              <DuplicatedFilesListItemText
+                displayStyle={TextDisplayStyle.Paragraph4}
+                color="secondary"
+                message={fileName}
+                key={`duplicated-files-list-item-text-${index}`}
```
Comment (OskarMichalkiewicz) [inline packages/app/src/files-manager/components/duplicated-files-modal/duplicated-files-modal.component.tsx:121]: I don't think that indexes are great keys, the file name would be a better choice i guess

Comment (lkowalczyk87) [reply packages/app/src/files-manager/components/duplicated-files-modal/duplicated-files-modal.component.tsx]: resolved in c93e17353a55ac2e1b9b8b9c4ea45d46e986cbdc

**Review by OskarMichalkiewicz — APPROVED** [review]

**Review by dkarski — APPROVED** [review]

**Pull request merged**